server/backends/disk_cache/disk_cache.go

```diff
@@ -120,7 +120,8 @@ func (c *DiskCache) initializeCache() error {
 		return err
 	}
 	c.mu.Lock()
```

Comment:
Is this block necessary? It looks like c.diskIsMapped is already guaranteed to be false here.

Reply:
nah, removed